### DISCUSSION TITLE
Feature Request: Ignore Conflict Option

### DISCUSSION THREAD

**Discussion by luxmanifestus** [X2CommunityCore/xcom2-launcher#319] [Ideas]
There currently exists the option to ignore missing dependencies. Can we extend the option to ignore conflicting installed mods? There are some conflicts that do not cause problems and the mod author advises one to ignore conflict indicators. It would be convenient to disable the red bars and notices for those specific conflicts.

**Comment by RevZero** [comment]
Hi and thanks for your input. I already created a ticket #226 for this some time ago. I just haven't found the time to look into it any further.
